ibft/storage/store.go

```diff
@@ -70,25 +72,37 @@ func (i *ibftStorage) CleanAllInstances(msgID []byte) error {
 	return nil
 }
 
-func (i *ibftStorage) UpdateParticipants(identifier convert.MessageID, slot phase0.Slot, newParticipants []spectypes.OperatorID) (updated bool, err error) {
+func (i *ibftStorage) UpdateParticipants(identifier convert.MessageID, slot phase0.Slot, newParticipants qbftstorage.Quorum) (updated bool, err error) {
 	i.participantsMu.Lock()
 	defer i.participantsMu.Unlock()
 
 	txn := i.db.Begin()
 	defer txn.Discard()
 
-	existingParticipants, err := i.getParticipants(txn, identifier, slot)
-	if err != nil {
-		return false, fmt.Errorf("get participants %w", err)
+	existing, err := i.getParticipantsBitMask(txn, identifier, slot)
+	if errors.Is(err, errNotBitmask) {
+		existingList, err := i.getParticipants(txn, identifier, slot)
+		if err != nil {
+			return false, fmt.Errorf("get participants: %w", err)
+		}
+
+		quorumAdapter := qbftstorage.Quorum{
+			Signers:   existingList,
+			Committee: newParticipants.Committee,
+		}
+
+		existing = quorumAdapter.ToSignersBitMask()
+	} else if err != nil {
+		return false, fmt.Errorf("get participants bitmask: %w", err)
```

Comment:
It's worth to mention is not backward compatible, once you update with this in your code, you cannot use your db with the old code. And it's not straight forward to get this because we're not doing any migration.
It might be smart to do something similar to what we did with the alan release and just fail if the version is older. @moshe-blox wdyt?